### DISCUSSION TITLE
Incorporating into Mongo-Express

### DISCUSSION THREAD

**Discussion by OOCAZ** [freecycle/free-swig#12] [General]
In order to get around some vulnerabilities with swig-templates mongo-express has switched over to using this. I am more than willing to contribute to keeping this alive. I am hopefully going to be updating old dependencies to newer versions soon. Let me know what else we might need to do in the future to sustain this package.

Have a great day!

**Comment by steevhise** [comment]
That's great, I hope this is still working for you. I'm no longer working at Freecycle and it seems as though my successors there don't have a clue that this package even exists. I also don't know how much time I will have to put in to it in the future but it's nice to know others are interested.

**Reply by steevhise** [reply]
I've just done another release because of a security issue. Looks like I still control the npm package so I will keep maintaining when I can.
